docs/01-app/05-api-reference/05-config/01-next-config-js/incrementalCacheHandlerPath.mdx

```diff
@@ -49,10 +45,27 @@ Returns `Promise<void>`.
 
 Returns `Promise<void>`. Learn more about [revalidating data](/docs/app/building-your-application/data-fetching/incremental-static-regeneration) or the [`revalidateTag()`](/docs/app/api-reference/functions/revalidateTag) function.
 
+### `resetRequestCache()`
+
+This method resets the temporary in-memory cache for a single request before the next request.
+
+Returns `void`.
+
 **Good to know:**
 
 - `revalidatePath` is a convenience layer on top of cache tags. Calling `revalidatePath` will call your `revalidateTag` function, which you can then choose if you want to tag cache keys based on the path.
 
+## Platform Support
+
+| Deployment Option                                                   | Supported         |
+| ------------------------------------------------------------------- | ----------------- |
+| [Node.js server](/docs/app/getting-started/deploying#nodejs-server) | Yes               |
+| [Docker container](/docs/app/getting-started/deploying#docker)      | Yes               |
+| [Static export](/docs/app/getting-started/deploying#static-export)  | No                |
+| [Adapters](/docs/app/getting-started/deploying#adapters)            | Platform-specific |
```

Comment:
nit: is it possible for a platform not to support users implementing a cache handler?